lib/dynamic_config/dcdo.rb

```diff
@@ -56,6 +56,7 @@ def frontend_config
       'student-email-post-enabled': DCDO.get('student-email-post-enabled', false),
       'progress-v2-metadata-enabled': DCDO.get('progress-v2-metadata-enabled', false),
       'show-updated-lms-content': DCDO.get('show-updated-lms-content', false),
+      'use-ghostscript-to-generate-pdfs': DCDO.get('use-ghostscript-to-generate-pdfs', false),
```

Comment:
I don't believe we need to add this flag to the subset of DCDO flags that get read by clientside code, since we're only ever reading it serverside.

Reply:
Hmm... I did a bit more digging on this to better understand what you're suggesting.  Paraphrasing what I believe to be true, give me a thumbs up if I have this right:

1. This code needed to configure DCDO flags in the front end / javascript
2. Since we will be only doing `DCDO.get` in back end code (ruby code), we don't need to add this DCDO flag here.
3. We will be able to still use DCDO flags when we merge this code into staging by doing `DCDO.set('use-ghostscript-to-generate-pdfs', true)` which will both create the flag, and set it to true.  The `ruby` file will be able to access this flag at this point.
4. Until that point, something like `DCDO.get('use-ghostscript-to-generate-pdfs', false)` will always return `false` because of the second parameter. 

That's the story I have woven together with some digging and context clues - let me know if that's all right.

Reply:
Yep, that's completely correct! Nice digging :smile: 

For some additional context; DCDO was originally exclusively serverside, and is still intended to be mostly used that way; the `frontend_config` functionality is an optional add-on.